src/draw/dma2d/lv_draw_dma2d.c

```diff
@@ -0,0 +1,569 @@
+/**
+ * @file lv_draw_dma2d.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw_dma2d.h"
+#if LV_USE_DRAW_DMA2D
+
+#include "../lv_draw_private.h"
+#include "../sw/lv_draw_sw.h"
+#include "../../misc/lv_area_private.h"
+
+#include "stm32h7xx_hal.h"
+#include "stm32h7xx_hal_dma2d.h"
+#include "stm32h7b3i_discovery_lcd.h"
```

Comment:
We need more generic header here. E.g. `F7`, `F4` and many other also have DMA2D. Dev board related headers (like `stm32h7b3i_discovery_lcd.h`) should be also avoided.